_posts/2019-04-25-subcontractors.markdown

```diff
@@ -0,0 +1,36 @@
+
+<!--more-->
+
+## Enter the subcontractor
+
+Ironically, for the same reasons that clients will hire an agency, agencies are sometimes in the market for contractors - or even other agencies.  If an agency and a freelancer are a good match for each other, the freelancer can allow the agency to tackle opportunities they otherwise couldn't, even if those opportunities don't justify a long term hire.
+
+As an added bonus, agencies willing to hire subcontractors expand the pool of agencies that could potentially meet a client's needs.  It's a win for everyone involved.
+
+## Potential pitfalls
+
+That's not to say that there aren't some downsides.  For one, take every downside of working remotely, and add an extra layer to that.  You can catch a lucky break if a subcontractor is in the same time zone as your client, for instance - but if they're not, then at best they'll have the same time zone problems that you might.  If they're in a different time zone then either you or your client, then obviously things can get difficult in a hurry.
+
+Speaking of time - finding contractors with the same availability and schedule as your needs can be remarkably hard.  Coordinating availability with your _client_ can be hard enough.  Coordinating it with a third party can be nearly intractable.
+
+Many of these issues can be fixed by having a deep pipeline of potential partners to work with.  Building that pipeline, however, brings difficulties of it's own - in addition to selling your services to clients, you now have the added work of selling your opportunities to potential partners.
+
+## Choosing a good partner
+
+As difficult as it can be - it's worth it.  If you're a development agency, a good subcontractor can be worth their weight in billable hours.  They can help you take advantage of opportunities you'd otherwise miss, and let you preview what it might be like to have another full time developer on staff.  Not to mention the value of building out the all important network, the list of contacts and colleagues that agencies often live or die by.  So what are some good attributes to look for?
+
+* *Previous experience with freelancing / contracting* - Working as a subcontractor is definitely not for someone who is just getting their feet wet in the freelancing world.
+* *Strong communication skills* - communication becomes even more critical with a third party in the mix.
+* *A pre-existing relationship* - while it's not strictly necessary, working with someone you already know (and therefore trust) can ease some of the difficulties.
+* *Flexibility* - subcontracting for a contracting agency is still an unusual practice. Having a bit of mental flexibility and the ability and willingness to try new things helps ensure success.
```

Comment:
@rdormer I'd add something about their technical skillset? Maybe if they are actively contributing to open source projects, publishing tech articles, or you've heard great things about their tech chops, then that's also a great thing.

It's certainly not the most important thing, but it's surely important too.

Reply:
I figured it would be a given that any potential contractor should have the requisite skill set.  Is there a specific reason to be explicit?

Reply:
@rdormer I think this should be explicitly written because there is certainly an advantage when the contractor has published content/code in the public domain (vs. someone who hasn't) 

It is not a deal breaker if they haven't, but it's one more point to be considered when interviewing a potential contractor or not. Anybody can say "I have been working with Ruby/Rails since 2009" but not everybody can say "Look, this